modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/AutoAnonymizeUserAssociatedDataEntityMVCActionCommand.java

```diff
@@ -51,6 +51,10 @@ protected void doProcessAction(
 
 		uadEntityAnonymizer.autoAnonymize(
 			getUADEntity(actionRequest, uadRegistryKey));
+
+		String redirect = ParamUtil.getString(actionRequest, "redirect");
+
+		sendRedirect(actionRequest, actionResponse, redirect);
```

Comment:
@willnewbury how are these `*MVCActionCommand` changes related to the kebab menu?

Reply:
Nevermind - I see you added the redirect as a parameter below